### ISSUE TITLE
Too low color contrast on URLs in CLI

### ISSUE THREAD

**Issue by boyum** [dequelabs/axe-cli#11]
Even though they are just a links, it would be nice to actually see where they go. Neither `#0000ff` on `#000000` (tested url) nor `#00006e` on `#000000` passes WCAG AA.

![image](https://cloud.githubusercontent.com/assets/9085189/24245210/31717d16-0fc2-11e7-8af6-29e2a8ccc9ca.png)

Using Command Prompt on Win10, default settings.

**Comment by WilcoFiers** [comment]
Right you are sir! I realised this when I was designing it. But look how pretty it is on mac ;)

![screenshot 2017-03-25 15 03 04](https://cloud.githubusercontent.com/assets/530687/24322864/574184c2-116c-11e7-8efc-095fa5076db0.png)

Seriously though, I'm glad you brought it up. I'm not sure I have a great solution. The thing is, people can set change the background color to whatever they want. What would you think of the following:

1. We change the color scheme to be different for windows than it is for mac. Assume black bg for windows, white for mac, and adjust the color accordingly. And then secondly
2. We provide a flag `--no-color` that allows users to turn off the colors all together so they can use whatever color scheme they have set up.

**Comment by boyum** [comment]
Thanks! Command Prompt forces me to copy the links to use them anyways, so as long as it's just the links being hard to read, I'll manage. The `--no-color` flag works great, but the colors help with readability, grouping the error messages.

Anyways, keep up the good work, it's a great tool!
